src/LookupTable/graphql/getDynamicSingleTable.query.ts

```diff
@@ -3,7 +3,7 @@ import { buildFieldList, capitalizeFirstLetter, toCamelCase } from '../utils'
 
 const getDynamicSingleTable = (structure: any) => gql`
     query getDynamicSingleTable {
-      lookupTable${capitalizeFirstLetter(toCamelCase(structure.name))}s {
+      lookupTable${capitalizeFirstLetter(toCamelCase(structure.name))} {
```

Comment:
Not a priority right now, but we should probably replace these with Lodash's case conversions, which we're already doing elsewhere. 

Then you can just use "StartCase" (I think) rather than chaining two methods like this

Reply:
Updating now...

Reply:
Hmmm turns out that lodash doesnt have a converter to that already... It also would need to use one function inside the other... So not worth it IMHO

Reply:
Hmm, I thought it did, I used something similar for Outcomes. Never mind, I'll change it at some point if I find it.

Reply:
It's called PascalCase, but yeah it looks like it's not in Lodash :(